net-utils/src/lib.rs

```diff
@@ -34,6 +34,11 @@ pub type PortRange = (u16, u16);
 pub const VALIDATOR_PORT_RANGE: PortRange = (8000, 10_000);
 pub const MINIMUM_VALIDATOR_PORT_RANGE_WIDTH: u16 = 17; // VALIDATOR_PORT_RANGE must be at least this wide
 
+#[allow(dead_code)]
```

Comment:
nit: You could probably annotate these constants with the same `#[cfg(not(any(windows, target_os = "ios")))]` as the function that uses them for consistency

Reply:
good call. updated!